pkg/cli/detect-iam-drift.go

```diff
@@ -92,12 +102,34 @@ func (c *DetectIamDriftCommand) Run(ctx context.Context, args []string) error {
 		"version", version.Version)
 
 	if c.flagOrganizationID == "" {
-		return fmt.Errorf("missing -organization-id")
+		return fmt.Errorf("missing --organization-id")
```

Comment:
This should actually be a single dash. We can do this in a follow-up PR though.

Reply:
Bah. Yeah. I'll update in a follow up